### PR TITLE
I implemented on FB registeration and Tour Scene.

### PR DESCRIPTION
**Pull request by thiago-19960626**

### CODE REVIEW

**Comment by awis0330** [comment]
#12

File: src/actions/register.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ var { Action, ThunkAction } = require('./types');
 
 function findByUsername(username: string){
     return new Promise((resolve, reject) => {
-        fetch(API_URL + `-public/users/?username=` + username, {
+        fetch(API_URL +`-public/users/?username=` + username, {
```
Comment (igaponov) [inline src/actions/register.js]: Why is this change here? Don't you use any linter to check your code before commit?

**Comment by thiago-19960626** [comment]
I think space inserted here

**Comment by igaponov** [comment]
No, the space was deleted here.

**Comment by thiago-19960626** [comment]
I think space was inserted between plus and dot

**Comment by igaponov** [comment]
Anyway it's better to avoid pointless changes that clog up history and consider using any linter as there are more than one developer works on this project.
